apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-site-editor-sidebar-clicks.ts

```diff
@@ -39,11 +40,16 @@ export const wpcomSiteEditorSidebarPatternsClick = () => {
 	};
 };
 
-export const wpcomSiteEditorSidebarStylesClick = () => {
+export const wpcomSiteEditorSidebarStylesClick = (): DelegateEventHandler => {
 	return {
 		id: 'wpcom_site_editor_sidebar_styles_click',
-		// \2f is the encoded slash.
-		selector: '#\\2fwp_global_styles',
+		/**
+		 * There is no id attribute (`#/wp_global_styles`) for themes that do not have style variations.
+		 *
+		 * @see https://github.com/okmttdhr/gutenberg/blob/70ccdefe1fd0b02638f5a8ae4f1c03e6f7fe27cc/packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js#L63-L83
+		 */
+		selector:
```

Comment:
This seems pretty fragile, but I can't think of a better way 😕 maybe @arthur791004 has some insights?

Reply:
Yes, it's fragile. Maybe the previous way with the translation might be better 🤔 

```js
target?.textContent !== __( 'Styles' )
```

Reply:
>target?.textContent !== __( 'Styles' )

Let me check if this works. If this is acceptable, 

https://github.com/Automattic/wp-calypso/pull/81000

Reply:
The text with the translation just worked!
https://github.com/Automattic/wp-calypso/pull/81002/commits/8be63010a99931242752529e75bd2019b80a7736

Reply:
Nice 👍